src/services/__tests__/QueueService.int.spec.ts

```diff
@@ -73,9 +79,20 @@ describe("QueueService tests", () => {
         expect(resultJoin).toHaveLength(1);
         expect(receivedQueueTime).toEqual(expectedQueueTime);
       });
-    });
+      it("In Active Match | Does not add to queue", async () => {
+        const mockMatchId = faker.datatype.uuid();
+        const mockPlayers = ActiveMatchBuilder.many(6, { matchId: mockMatchId });
+
+        const oneOfThePlayers = faker.random.arrayElement(mockPlayers);
```

Comment:
Do you need to build an active match if all you're doing is picking the id of one them? I think using a `faker.datatype.uuid()` in the `await joinQueue` call would give you the same effect.

Reply:
So I understand - I can pass in any id here because I mocked the repo and will always return true - so a team doesn't actually need to be created.  is that what you are saying?

Reply:
Yepp! None of the actual logic in ActiveMatch repository is being used since we're just telling the function what to return. So we don't actually need an active match for the mock to do it's thing.

Reply:
Cool.  Fixed in my next commit.